interactive_templates/templates/v2/analysis/plot_measures.py

```diff
@@ -21,6 +22,15 @@ def main():
     df = pd.read_csv(
         f"{ args.output_dir }/joined/measure_all.csv", parse_dates=["date"]
```

Comment:
Might it be more flexible to pass as an argument the path to the input file separate from the output_dir? What if measure_all does not share the same path root as where you want to put the output?